tsdb/engine/tsm1/compact.go

```diff
@@ -255,7 +255,7 @@ func (c *DefaultPlanner) ParseFileName(path string) (int, int, error) {
 
 // FullyCompacted returns true if the shard is fully compacted.
 func (c *DefaultPlanner) FullyCompacted() (bool, string) {
-	gens := c.findGenerations(false)
+	gens := c.findGenerations(true)
```

Comment:
@devanbenz I think the code works properly for compactions, but I think we should find a better name for `FullyCompacted`, since it will now return true even if there compactions currently running. I checked and there are no `FullyCompacted` calls in plutonium, so the change should not ripple up.

Not sure what a good name is, though. Maybe `OptimizationsAvailable`?

Reply:
Changed the name to `CompactionOptimizationAvailable`